_layouts/translated.html

```diff
@@ -1,17 +1,16 @@
 ---
 layout: default
 ---
+
 {% if page.namespace %}
-  <div align="right">
-    {% assign languages = page.languages | default: site.languages %}
-    {% for lang in languages %}
-      {% if languages.size == 1 %}
-      {% elsif site.lang == lang %}
-        <b>{% t langs.{{ lang }} %}</b>
-      {% else %}
-        <a href="{% tl {{ page.namespace }} {{ lang }} %}" >{% t langs.{{ lang }} %}</a>
-      {% endif %}
-    {% endfor %}
-  </div>
-{% endif %}
-{{ content }}
+<div align="right">
+  {% assign languages = page.languages | default: site.languages %} {% for lang
+  in languages %} {% if languages.size == 1 %} {% elsif site.lang == lang %}
```

Comment:
This one we can revisit in another PR too - there has to be another way for ERB to make it easier to maintain etc.

Reply:
I needed to add an override so the prettier liquid plugin could apply to .html extensions